client/src/views/OrgsList.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import { NavLink } from 'react-router-dom';
+import Typography from '@mui/material/Typography';
+import makeStyles from '@mui/styles/makeStyles';
+import Card from '@mui/material/Card';
+import TodayOutlined from '@mui/icons-material/TodayOutlined';
+import RoomOutlined from '@mui/icons-material/RoomOutlined';
+import Grid from '@mui/material/Grid';
+
+import type { Theme } from '@mui/material/styles';
+
+import type { Asset } from '../types';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  needsAndOffersSub: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    flexWrap: 'wrap',
+    alignItems: 'stretch',
+  },
+  card: {
+    marginBottom: '30px',
+    '&:not(:last-child)': {
+      marginRight: '20px',
+    },
+    flex: '0 0 30%',
+  },
+  cardImg: {
+    borderRadius: '5px',
+    margin: '10%',
+    maxWidth: '80%',
+  },
+  needsAndOffersHeader: {
+    textAlign: 'left',
+    paddingBottom: '20px',
+  },
+  needsAndOffers: {
+    padding: '10%',
+  },
+  cardText1: {
+    padding: '0 10%',
+  },
+  cardText2: {
+    padding: '0 10% 10%',
+  },
+}));
+
+type Props = {
+  assets: Asset[];
+  headerContentRight?: JSX.Element;
+  headerText: string;
+};
+
+function AssetsList(props: Props): JSX.Element {
+  const classes = useStyles();
+  const { assets, headerContentRight, headerText } = props;
+  if (!assets) return <> </>;
+
+  return (
+    <>
+      <Grid container item justifyContent="space-between">
+        <Typography variant="h4" component="h4" className={classes.needsAndOffersHeader}>
+          {headerText}
+        </Typography>
+        {headerContentRight ?? null}
+      </Grid>
+      <div className={classes.needsAndOffersSub}>
+        {assets.map((asset) => (
+          <NavLink to={`/asset/${asset.id}`} key={asset.id} className={classes.card}>
```

Comment:
to={`/organizations/${org.id}`

Reply:
plz fix

Reply:
OrgsList.tsx
![Screenshot 2022-11-04 at 7 29 58 PM](https://user-images.githubusercontent.com/92830192/200097190-aa2c2986-cba0-4e9c-9947-e331ea9aafa5.png)

I was looking at commit [6fa2bbc](https://github.com/Nonprofit-Exchange-Hub/web-app/pull/230/commits/6fa2bbc334d5b86258dfed19363d727991761c32) 8 days ago. Let me know if you see something else and which commit it is. I'll fix it if so.